docs/content/Schema/cube.md

```diff
@@ -100,11 +100,40 @@ For previous example following query
 leads to a join
 
 ```sql
-select COALESCE("users".name, "contacts".name) "users__name" 
+select COALESCE("users".name, "contacts".name) "users__name"
 FROM users "users"
 LEFT JOIN contacts "contacts" ON "users".contact_id = "contacts".id
 ```
 
+## Abstract cubes
+
+Abstract cubes can be defined by simply omitting the first parameter to the
+`cube()` function. Cubes defined in this way can still be extended from, but
```

Comment:
Can we remove "from" so it says "can still be extended, but..."?

Reply:
Yep makes sense, good catch 😄